services/idp/package.json

```diff
@@ -153,5 +153,10 @@
     "webpack": "4.47.0",
     "webpack-manifest-plugin": "4.1.1",
     "workbox-webpack-plugin": "7.0.0"
+  },
+  "pnpm": {
+    "overrides": {
+      "kpop>cldr": ""
```

Comment:
Could you please add a comment or link this pr for now ?

Reply:
Douglas Crockford: 
> I removed comments from JSON because I saw people were using them to hold parsing directives, a practice which would have destroyed interoperability. I know that the lack of comments makes some people sad, but it shouldn't.

> Suppose you are using JSON to keep configuration files, which you would like to annotate. Go ahead and insert all the comments you like. Then pipe it through JSMin before handing it to your JSON parser.﻿


TL;DR: no comments in json. Vscode and TS allow it, but it's non-standard and pnpm doesn't.

Reply:
That's in fact why I didn't add a comment :-)